### PR TITLE
"Portfolio: Setup

### PR DESCRIPTION
**Pull request by ManarOmari**
I closed the previous pull request by mistake

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ https://github.com/ManarOmari/Portoflio/tree/main
 
 ## Getting Started
 
-
+started to buils mobile sketon for a Protoflio template 
+first : header contains a logo and tool bar 
+then set background image for the content section which conains a text and images
```
Comment (koachcode) [inline README.md]: - Please add the CSS styling to your project as it is missing that, making your project not look like the Figma design 
##### Figma design
Screenshot from 2022-05-27 06-46-53

##### Your design
Screenshot from 2022-05-27 06-43-19

**Notes:**
- Please also make sure you are done with the given sectioned task, before requesting a review
- Please also make sure you use only one branch when neccessary

If you need for explanation concerning this feedback, Feel free to reach out to me on Slack: `@Kanu mike chibundu`

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @ManarOmari , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- Linter Passed :+1: 


 ## Required Changes ♻️
_Check the comments under the review._

Reach me on Slack  `@Kanu mike chibundu`

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ManarOmari, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] Kindly make sure to follow the loom video attached to make changes [view here](https://www.loom.com/share/15243cf596d34ed89975bb1bd3d62099).
- [ ] Optional (But highly recommended): Kindly make sure to Name your Pull request title "Portfolio: Setup" and Pull request description to "Made the first milestone for the portfolio by the building the introduction part as per the Figma design."

As suggested previously and shown in the video attached kindly:
- [ ] Add the background image correctly as per the Figma design.
- [ ] Kindly follow the spacing and font size as per the Figma design.
- [ ] Kindly display the heading with the correct font weight as per the Figma design.
- [ ] Make sure to display all the contact image icons as per the Figma design.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ManarOmari** [comment]
Made the first milestone for the portfolio by the building the introduction part as per the Figma design

**Comment by ManarOmari** [comment]
@sja-thedude I think I fixed the errors and modified all the required issues. Please, can you let me if I should to request review ?

**Comment by m05e5** [comment]
Hi please, there are still things to improve. As mentioned by the previous reviewers, you don't have a background image and you have nav links instead of the hamburger menu for the mobile version

File: index.html [hunk 2]
```diff
@@ -27,43 +24,46 @@
...
+           <div class="anglelist">
             <a href="#">
-                <img src="/assets/images/anglelist.png" alt="angle list in  image" class="anglelist">
+                <img src="assets/images/anglelist.png" alt="angle list in  image" class="anglelist">
                </a>
-           
-       </div>
-    </div>
+           </div>
+            </div>
```
Comment (m05e5) [inline index.html]: - _[optional]_ I recommend using `<ul>` for the list of social media icons and `li` for each icon

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @ManarOmari , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

**imp**
image
- [ ] Please in place of nave links kindly display the hamburger menu
- [ ] Your text is overflowing the screen please fix that
- [ ] kindly add a background image as it is on the design
- [ ] kindly readjust your padding so it resemble the Figma design
_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ManarOmari** [comment]
Hello ,
I finished the project part 1 and I misunderstood what the design I will to
do , all a week i was working on desktop design like this one

**Comment by ManarOmari** [comment]
I updated the design to mobile version since I was working on desktop design all the last week . this is why i didnt go forward . @m05e5

File: index.html [hunk 3]
```diff
@@ -4,66 +4,67 @@
...
-                     <li><a href="#about.html">About</a></li>
-                     <li><a href="#Contact.html">Contact</a></li>
-                     </ul>
-                    </section>
-               </div>
+           
+             <nav>
+                 <div class="logo">
+                   <h1><a href="#">My Logo</a></h1>
+                   <a class="hamburger-menue" href="#">
```
Comment (danielufeli) [inline index.html:17]: - [Optional] Nicely done, kindly add a little `right-margin` to the hamburger menu so it aligns with what you have in the Figma Design

**Review by danielufeli — APPROVED** [review]
Hi @ManarOmari ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**